agents/World.java

```diff
@@ -138,4 +151,51 @@ public Country getCountry() {
             return country;
         }
     }
+
+    private class Loggable {
+
+        private BufferedWriter logWriter;
+
+        Loggable(String logName) {
+            if (logDirPath != null) {
+                Path path = logDirPath.resolve(logName + ".txt");
+                try {
+                    logWriter = new BufferedWriter(new OutputStreamWriter(
+                            new FileOutputStream(path.toString()), "utf-8"));
+                } catch (IOException e) {
+                    e.printStackTrace();
+                }
+            }
+        }
```

Comment:
We don't need a Logger class, we can just use Logback, and you can create a simple program to split the log file output (which is what this class seems to enable).

Reply:
I propose that in a subsequent PR we remove the logging and implement it properly as you suggest.